drivers/entropy/entropy_stm32.c

```diff
@@ -487,6 +487,18 @@ static int entropy_stm32_rng_init(const struct device *dev)
 		(clock_control_subsys_t *)&dev_cfg->pclken);
 	__ASSERT_NO_MSG(res == 0);
 
+
+#if DT_INST_NODE_HAS_PROP(0, health-test-config)
```

Comment:
instead of relying on the DTS, checking the device and the LL_RNG_SetHealthConfig is defined
```
#ifdef (RNG_HTCR)
LL_RNG_SetHealthConfig
```

Reply:
This is not possible:
RNG_HTCR doesn't exist, but rather RNG->HTCR.
Some series define RNG_HTCR_HTCFG, but not all series, so it is not usable as generic mechanism.
And LL_RNG_SetHealthConfig() is a function not a macro, so it cannot be used for preprocessing.
So I rely on DTS. Up to user to set it correctly.